CTestRun.cmake

```diff
@@ -5,6 +5,7 @@ set(CTEST_SOURCE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 
 if(NOT "${GITHUB_PR_ID}" STREQUAL "")
   set(CTEST_CHANGE_ID ${GITHUB_PR_ID})
+  set(CTEST_BUILD_NAME "${CTEST_BUILD_NAME}-PR${GITHUB_PR_ID}")
```

Comment:
Isn't that variable set externally L11 in the configuration file?

Reply:
It's overwritten later but shouldn't be there in the first place.

Reply:
Fixed.